src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -92,24 +92,37 @@ class oopDesc;
 
 #define BOOL_TO_STR(_b_) ((_b_) ? "true" : "false")
 
+// Format 8-bit quantities.
```

Comment:
Maybe a little comment explaining the various suffixes?

Reply:
Sure

Reply:
Added a comment. I see that there's some inconsistency, with the names used for SIZE_FORMAT_HEX. And we could probably replace PTR32_FORMAT/PTR64_FORMAT/INTPTRNZ_FORMAT, now that we have these new integer format specifiers. I'd like to do that cleanup as a separate patch.

Reply:
Hmm. I found some more inconsistencies. I think at this point, It would be better if I just unified all the format specifier macros, and rename this bug.